deployment/docker-compose/docker-compose.yml

```diff
@@ -57,6 +57,8 @@ services:
       - .env
     volumes:
       - ./litellm_proxy_config.yaml:/app/config.yaml
+    ports:
+      - "4000:4000"
```

Comment:
Why are we exposing this? It's only used internally right?

Reply:
oops, left over from testing I think. removed.